hack/test-cluster-class.sh

```diff
@@ -0,0 +1,39 @@
+#/bin/bash
+
+ROOT_DIR=$(cd $(realpath $(dirname $0))/.. && pwd)
+
+if [[ ! -d "./output" ]]; then
+    echo "create output directory"
+    mkdir "./output"
```

Comment:
if [[ ! -d "${ROOT_DIR}/output" ]]...
mkdir ${ROOT_DIR}/output

same for if in line 10

Reply:
fixed